### ISSUE TITLE
Are resume_markers supposed to be newly created all the time?

### ISSUE THREAD

**Issue by jorisw** [carlos8f/zenbot#2523]
### System information
- **Have I written custom code (as opposed to using zenbot vanilla)**: Yes, lots

- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: macOS 11.0.1

- **Zenbot version** (commit ref, or version): `f91355a`

- **Zenbot branch**: `unstable`

- **NodeJS version**: 14

- **Exact command to reproduce (include everything)**: `zenbot trade binance.ETH-USDT --strategy my-strategy`

- **Did I make any changes to conf-sample.js?**: Just API keys.

### Describe the problem

My database contains about 500K resume markers, while I'm using under 100 assets. That said, I have started and stopped many many sessions.

Are entries in the `resume_markers` collection, supposed to pile up this way, or did I break something that causes existing ones not to be updated?

In other words, are the continued newly created resume_markers by design?

**Comment by yellowgh0st** [comment]
Offtopic// 
@jorisw Hey, do you think we can revive this? I saw you're pretty active and seems you are using it with such a heavy setup.
_I was trying to find a good framework for algo trading, but this is still far the best what we have and it sorta works._
I'd like to start fixing the stuff and maybe lift off some stuff, but I'm unsure where to start because most stuff reported is hard to reproduce for me. Do you have any other issues which would be worth collecting and looking into?

**Comment by DeviaVir** [comment]
Sounds like a bug.

**Comment by jorisw** [comment]
@yellowgh0st As much as I'd like to help revive this project, I am a mere user like the rest of us here. I just got in the mood to answer some of the issues right after I filed mine.

I agree with your classification of zenbot as something that sorta works, and I don't know of many other projects that are either feature complete or still maintained. I've looked at a few on GitHub, and sooner or later discovered they weren't better or worse than zenbot in terms of completeness or activity on the project. Just kinda seems like most authors give up after a while.

I can also see how it's hard to identify what to work on, as far as zenbot goes, looking at the giant pile of issues still open. My only suggestion would be to try and use zenbot yourself, see where it takes you, and fix whatever problem you encounter. I believe different users are trying to accomplish very different things. So if you're going to invest your time, by all means, make sure it benefits your own objectives. (#2404 could use some love though, it seems to be a blocker for some users.)

I must warn you that the general code quality of zenbot is very low, from badly designed, to untested, to riddled with incomprehensible naming conventions. In the parts I've added for myself, I've separated my code from zenbot's as much as possible.
